tlatools/org.lamport.tlatools/test/tlc2/util/BufferedRandomAccessFileTest.java

```diff
@@ -80,8 +77,8 @@ public void testWriteSeekNoLength() throws IOException {
 			}
 		} catch (IOException expected) {
 			return;
-		} catch (Exception e) {
-			fail(e.getMessage());
+//		} catch (Exception e) {
```

Comment:
Was this never necessary?  I haven't been able to figure out why this was ever here in the first place :laughing:

Reply:
It was not. 😆 It does exactly one thing, which is to hide the stack trace from the thrown exception `e`. That one thing is bad, so I've removed these catch blocks.